### DISCUSSION TITLE
KRT-2 Interface

### DISCUSSION THREAD

**Discussion by hvennekate** [XCSoar/XCSoar#1592] [Q&A]
Hi community!
I hope this is the right place to ask this question: In https://github.com/XCSoar/XCSoar/blob/3cbf18270d1efdd3579bc11f709b4364ac3ef7d4/src/Device/Driver/KRT2.cpp#L25 there is a reference to a document of the KRT-2 interface. I am looking for such documentation for a different project and unfortunately the link no longer works.
Would anybody happen to have any pointers where that document could still be found?
Many thanks in advance!

**Comment by lordfolken** [comment]
I still have a backup of the former bug tracker. I should have it'

**Comment by lordfolken** [comment]
https://github.com/XCSoar/website/issues/122

**Reply by hvennekate** [reply]
So it would eventually end up on the website? That would be great. Many thanks in advance!

**Comment by F-Phi** [comment]
you can contact TQ as well, they provide the document.
https://www.tq-group.com/de/unternehmen/kontakt/

**Reply by hvennekate** [reply]
Sorry to not have come back to this earlier: I did and they were very forthcoming and sent three documents. If you want to collect them somewhere, I'd be more than happy to provide them, but I guess we'd have to ask them for permission first.

**Reply by lordfolken** [reply]
If you could ask that would be great. that way we have truly open source docs.

**Reply by hvennekate** [reply]
So, the situation is as follows: getting permission to release the documents in question to a larger audience would be difficult. However, they can be made available upon request to TQ. I did, however, receive an older document from the time when Dittel was still the company manufacturing the KRT-2 that would be ok for broader release. See the attachment.
[Interface_Ext_V18.pdf](https://github.com/user-attachments/files/17593251/Interface_Ext_V18.pdf)
